core/modules/file/file.install

```diff
@@ -105,11 +105,23 @@ function file_schema() {
         'not null' => TRUE,
         'default' => 0,
       ),
+      'type' => array(
+        'description' => 'The type of this file.',
+        'type' => 'varchar',
+        'length' => 50,
+        'not null' => TRUE,
+        // If the FILE_TYPE_NONE constant ever changes, then change the value here
+        // too, and add an update function to deal with existing records. The
+        // constant isn't used here, because there may be cases where this function
+        // runs without the module file loaded.
+        'default' => 'undefined',
+      ),
```

Comment:
We need another update hook that adds this new column, and the new `file_type` index to this table.

Reply:
re-numbered the updates, running test update again now.